src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -56,6 +56,13 @@ sealed public class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
 
         private IDebugSettingsCallback110 _settingsCallback;
 
+        public static List<int> ChildProcessLaunch;
```

Comment:
Since there is no locking on List, I would probably suggest either adding an accessor method instead of making the collection public.